vm/src/obj/objrange.rs

```diff
@@ -106,22 +107,9 @@ pub fn init(context: &PyContext) {
                      These are exactly the valid indices for a list of 4 elements.\n\
                      When step is given, it specifies the increment (or decrement).";
 
-    extend_class!(context, range_type, {
-        "__bool__" => context.new_rustfunc(PyRange::bool),
-        "__contains__" => context.new_rustfunc(PyRange::contains),
+    extend_class!(context, &context.range_type, {
         "__doc__" => context.new_str(range_doc.to_string()),
```

Comment:
The macros can also autogenerate `__doc__` from Rust doctrings now.

Reply:
And you can put `__new__` in the pyimpl as well.

Reply:
Thanks, I will try

Reply:
Fixed them